client/src/components/register.js

```diff
@@ -94,9 +95,33 @@ class RegisterForm extends React.Component {
     })
   }
 
-  handleSubmit(event) {
+  async handleSubmit(event) {
     event.preventDefault()
-    console.log(this.state)
+
+    const email = this.state.form.email.value !== ''
```

Comment:
you might be able to get away with just `const email = this.state.form.email.value ? blah : blah` without having to check if its !== '', because I believe empty string is falsy. See if it'll work like this, if so, that's one less comparison the app has to do: +1 Performance.